src/domains/ioweb-app/06_function_ioweb_profile.tf

```
@@ -0,0 +1,308 @@
+###
+### SECRETS
+###
+data "azurerm_key_vault_secret" "api_beta_testers" {
+  name         = "ioweb-profile-api-beta-testers"
+  key_vault_id = data.azurerm_key_vault.kv.id
+}
+
+data "azurerm_key_vault_secret" "functions_fast_login_api_key" {
+  name         = "ioweb-profile-functions-fast-login-api-key"
+  key_vault_id = data.azurerm_key_vault.kv.id
+}
+
+data "azurerm_key_vault_secret" "functions_app_api_key" {
+  name         = "ioweb-profile-functions-app-api-key"
+  key_vault_id = data.azurerm_key_vault.kv.id
+}
+
+data "azurerm_key_vault_secret" "spid_login_jwt_pub_key" {
+  name         = "spid-login-jwt-pub-key"
+  key_vault_id = data.azurerm_key_vault.kv.id
+}
+
+data "azurerm_key_vault_secret" "spid_login_api_key" {
+  name         = "ioweb-profile-spid-login-api-key"
+  key_vault_id = data.azurerm_key_vault.kv.id
+}
+###
+
+locals {
+  function_ioweb_profile = {
+    app_settings = {
+      FUNCTIONS_WORKER_PROCESS_COUNT = 4
+      NODE_ENV                       = "production"
+
+      // Keepalive fields are all optionals
+      FETCH_KEEPALIVE_ENABLED             = "true"
+      FETCH_KEEPALIVE_SOCKET_ACTIVE_TTL   = "110000"
+      FETCH_KEEPALIVE_MAX_SOCKETS         = "40"
+      FETCH_KEEPALIVE_MAX_FREE_SOCKETS    = "10"
+      FETCH_KEEPALIVE_FREE_SOCKET_TIMEOUT = "30000"
+      FETCH_KEEPALIVE_TIMEOUT             = "60000"
+
+      // --------------
+      // FF AND TESTERS
+      // --------------
+      FF_API_ENABLED = "BETA"
+      BETA_TESTERS   = data.azurerm_key_vault_secret.api_beta_testers.value
+
+      // ------------
+      // JWT Config
+      // ------------
+      BEARER_AUTH_HEADER       = "authorization"
+      EXCHANGE_JWT_ISSUER      = "api-web.io.pagopa.it/ioweb/auth"
```

This token is issued by the function itself
```suggestion
      EXCHANGE_JWT_ISSUER      = "api-web.io.pagopa.it/ioweb/backend"
```

---

fixed in https://github.com/pagopa/io-infra/pull/678/commits/7d0365144e7607bc7d14a5e2add1fc3ce10d055d